readme.md

```diff
@@ -46,6 +46,17 @@ $ curl -sSLO https://github.com/sindresorhus/run-node/raw/master/run-node && chm
 ```
 ./run-node file.js
 ```
+### Add and edit customizable cache path and error message
+
+The cache path and error message are defined by `RUN_NODE_CACHE_PATH` and `RUN_NODE_ERROR_MSG` respectively. To add these variables to `~.bashrc` or `~/.bash_profile`. This variables can also be used in scripts that want to use set a different cache path or message.
+
+Default configuration:
+
+```
+#node-run variables
```

Comment:
#node-run I think this line is not necessary and @sindresorhus already asked to remove it.

Reply:
Yes that's true. Sorry, I didn't get it the first time... Now I understood :+1: line removed